File: Armstrongbtwninterval/armstrongbtwin2numbers.java
import  java.util.Scanner;
public class armstrongbtwin2numbers{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter Range between number you want to see:");
        int l = sc.nextInt();
        int h = sc.nextInt();
        System.out.println("Numbers between "+l+" and "+h);
        for(int num=l;num<=h;num++){
            int s=0,d,temp;
            int len = getOrder(num);
            temp=num;
            while(temp!=0){
                d=temp%10;
                s=s+(int)Math.pow(d,len);
                temp/=10;
            }
            if(num==s){
                System.out.print(s+" ");
        }

    }
}
static int getOrder(int num){
    int l=0;
    while(num!=0){
        l++;
        num/=10;
    }
    return l;
}
}

File: Armstrongbtwninterval/armstrongbtwin2numbers.py
print("Enter range between you want to see all armStrong numbers")
l =int(input("Enter first number"))
h = int(input("Enter Second number"))
def getOrder(num):
    l=0
    while(num!=0):
        l+=1
        num//=10
    return l
for num in range(l,h+1):
    temp=num
    s=0
    len=getOrder(num)
    while(temp!=0):
        d=temp%10
        s+=d**len
        temp//=10
    if(num==s):
        print(s,end=" ")

        